site/docs/master/httpproxy.md

```diff
@@ -355,7 +355,7 @@ For `header` conditions there is one required field, `name`, and five operator f
 
 - `exact` is a string, and checks that the header exactly matches the whole string. `notexact` checks that the header does *not* exactly match the whole string.
 
-#### Multiple Routes
+### Routes
 
 HTTPProxy must have at least one route or include defined.
```

Comment:
This is out of scope for this review, but since I saw it I don't think this is true, a httpproxy can be valid without a routes or includes section if it has a tcpproxy section.

Reply:
Separate issue for this?

Reply:
#1988